docs/src/DeveloperDocumentation/release.md

```diff
@@ -40,6 +40,9 @@ For OSCAR 1.0.0 we plan to have one or two release candidates but unlike julia a
 
 ## Releases
 
+Just before making the release, `CHANGELOG.md` should be updated.
+See [Updating `CHANGELOG.md`](@ref) for details.
+
 Once the version is at X.Y.0 the version can be registered in the julia registry with `@JuliaRegistrator register()`. In this case TagBot will create the corresponding release, but preferably recheck and clean up the list of changes since unfortunately TagBot currently does not take the branch into account and will show all PRs that have been merged to master.
```

Comment:
```suggestion
Once the version is at X.Y.0, the version can be registered in the julia registry with `@JuliaRegistrator register()`. In this case TagBot will create the corresponding release, but preferably recheck and clean up the list of changes since unfortunately TagBot currently does not take the branch into account and will show all PRs that have been merged to master.
```